spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -239,6 +239,16 @@ def price_tier
         client.price_tier(self.apple_id)
       end
 
+      # set the availability. This method doesn't require `save` to be called
+      def update_availability!(territories, opts = {})
```

Comment:
Thanks for the review! I completely see where you are coming from and like that approach.